vite.config.ts.timestamp-1667858225324.mjs

```diff
@@ -0,0 +1,13 @@
+// vite.config.ts
+import { defineConfig } from 'file:///home/dev/Documents/code/arns-react/node_modules/vite/dist/node/index.js';
+import react from 'file:///home/dev/Documents/code/arns-react/node_modules/@vitejs/plugin-react/dist/index.mjs';
+import svgr from 'file:///home/dev/Documents/code/arns-react/node_modules/vite-plugin-svgr/dist/index.mjs';
+var vite_config_default = defineConfig({
+  plugins: [svgr(), react()],
+  base: '/arns-react/',
+  define: {
+    APP_VERSION: JSON.stringify(process.env.npm_package_version),
+  },
+});
+export { vite_config_default as default };
+//# sourceMappingURL=data:application/json;base64,ewogICJ2ZXJzaW9uIjogMywKICAic291cmNlcyI6IFsidml0ZS5jb25maWcudHMiXSwKICAic291cmNlc0NvbnRlbnQiOiBbImNvbnN0IF9fdml0ZV9pbmplY3RlZF9vcmlnaW5hbF9kaXJuYW1lID0gXCIvaG9tZS9kZXYvRG9jdW1lbnRzL2NvZGUvYXJucy1yZWFjdFwiO2NvbnN0IF9fdml0ZV9pbmplY3RlZF9vcmlnaW5hbF9maWxlbmFtZSA9IFwiL2hvbWUvZGV2L0RvY3VtZW50cy9jb2RlL2FybnMtcmVhY3Qvdml0ZS5jb25maWcudHNcIjtjb25zdCBfX3ZpdGVfaW5qZWN0ZWRfb3JpZ2luYWxfaW1wb3J0X21ldGFfdXJsID0gXCJmaWxlOi8vL2hvbWUvZGV2L0RvY3VtZW50cy9jb2RlL2FybnMtcmVhY3Qvdml0ZS5jb25maWcudHNcIjtpbXBvcnQgeyBkZWZpbmVDb25maWcgfSBmcm9tICd2aXRlJztcbmltcG9ydCByZWFjdCBmcm9tICdAdml0ZWpzL3BsdWdpbi1yZWFjdCc7XG5pbXBvcnQgc3ZnciBmcm9tICd2aXRlLXBsdWdpbi1zdmdyJztcblxuLy8gaHR0cHM6Ly92aXRlanMuZGV2L2NvbmZpZy9cbmV4cG9ydCBkZWZhdWx0IGRlZmluZUNvbmZpZyh7XG4gIHBsdWdpbnM6IFtzdmdyKCksIHJlYWN0KCldLFxuICAvLyBUT0RPOiByZW1vdmUgdGhpcyB3aGVuIG5vIGxvbmdlciBkZXBsb3lpbmcgdG8gR0ggcGFnZXNcbiAgYmFzZTogJy9hcm5zLXJlYWN0LycsXG4gIGRlZmluZToge1xuICAgIEFQUF9WRVJTSU9OOiBKU09OLnN0cmluZ2lmeShwcm9jZXNzLmVudi5ucG1fcGFja2FnZV92ZXJzaW9uKSxcbiAgfSxcbn0pO1xuIl0sCiAgIm1hcHBpbmdzIjogIjtBQUEyUixTQUFTLG9CQUFvQjtBQUN4VCxPQUFPLFdBQVc7QUFDbEIsT0FBTyxVQUFVO0FBR2pCLElBQU8sc0JBQVEsYUFBYTtBQUFBLEVBQzFCLFNBQVMsQ0FBQyxLQUFLLEdBQUcsTUFBTSxDQUFDO0FBQUEsRUFFekIsTUFBTTtBQUFBLEVBQ04sUUFBUTtBQUFBLElBQ04sYUFBYSxLQUFLLFVBQVUsUUFBUSxJQUFJLG1CQUFtQjtBQUFBLEVBQzdEO0FBQ0YsQ0FBQzsiLAogICJuYW1lcyI6IFtdCn0K
```

Comment:
why is this being added?

Reply:
i believe thats a file thats created during dev server running. ussually it removes it on its own, must have bugged when I last shut down the server. removed it.